### DISCUSSION TITLE
Uploading issue

### DISCUSSION THREAD

**Discussion by steve-mcneil** [wayne-love/sn_esp32#66] [Q&A]
Hi everyone, firstly great project, looking forward to get everything up an running on our spa, slowly including more of the house in home assistant.

I'm having an issue uploading the bin to my esp32 in vscode that hopefully has a straightforward answer. To given some background I've dabbled in embedded stuff over the years, and am reasonably comfortable in the arduino ide, but vscode and platformio are new to me.

I'm able to successfully build the repo straight out of git, yet when I get to uploading the bin I get an error stating that I have an esp32s, not an esp s3. I've tried changing the board in platformio.ini under [env:spa-control-pcb] from  "esp32-s3-devkitc-1" to "esp32dev" and am now getting an error regarding the serial port not being defined.

I'm probably doing something stupid but happy to hear suggestions or guidance.

Cheers,
Stephen

**Comment by lucasnz** [comment]
It may be easiest to discuss if join us on discord here: https://discord.gg/t3FzPtKB

What board do you have? Can you send a link showing the board?
